src/ngx_http_lua_headers_in.c

```diff
@@ -645,14 +645,95 @@ ngx_http_clear_builtin_header(ngx_http_request_t *r,
 }
 
 
+static ngx_hash_init_t  lua_req_header_hash;
+static u_char           header_string[512];
```

Comment:
1.Yes, header_string can use local variable instead
2.The question here is that  where to store lua_req_header_hash.hash?